src/System.Linq/tests/ShortCircuitingTests.cs

```diff
@@ -111,6 +111,94 @@ public void MinNullableSingleDoesntCheckAll()
             Assert.Equal(5, tracker.Moves);
         }
 
+        [Fact]
+        public void MinDoubleDoesntCheckAllStartLeadingWithNaN()
+        {
+            var tracker = new TrackingEnumerable(10);
+            var source = tracker.Select(i => i == 1 ? double.NaN : (double)i);
+            Assert.True(double.IsNaN(source.Min()));
+
+            //see https://github.com/dotnet/corefx/issues/38392.
```

Comment:
These comments linking to the issue aren't necessary.  You can remove them.

Reply:
@stephentoub OK, Thanks for your review.